src/condition_variable.hpp

```diff
@@ -183,7 +183,7 @@ class condition_variable_t
     {
         pthread_condattr_t attr;
         pthread_condattr_init (&attr);
-#ifndef ZMQ_HAVE_OSX
+#if !defined(ZMQ_HAVE_OSX) || (defined(ZMQ_HAVE_ANDROID) && !defined(HAVE_PTHREAD_COND_TIMEDWAIT_MONOTONIC))
         pthread_condattr_setclock (&attr, CLOCK_MONOTONIC);
```

Comment:
This is still failing the android build job. Maybe the condition is missing some brackets?